components/script/bluetooth_blacklist.rs

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use regex::Regex;
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+use std::string::String;
+
+const EXCLUDE_READS: &'static str = "exclude-reads";
+const EXCLUDE_WRITES: &'static str = "exclude-writes";
+const FILE_PATH: &'static str = "./resources/gatt_blacklist.txt";
+const VALID_UUID_REGEX: &'static str = "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}";
+
+thread_local!(pub static BLUETOOTH_BLACKLIST: RefCell<Option<BluetoothBlacklist>> =
+              RefCell::new(BluetoothBlacklist::init()));
+
+pub struct BluetoothBlacklist(HashMap<String, ValueType>);
+
+#[derive(Eq, PartialEq)]
```

Comment:
because of this:
```
   Compiling script v0.0.1 (file:///home/zakor1gyula608/servotimeout/components/script)
/home/zakor1gyula608/servotimeout/components/script/bluetooth_blacklist.rs:23:1: 23:14 error: the trait bound `bluetooth_blacklist::ValueType: core::cmp::PartialEq` is not satisfied [E0277]
/home/zakor1gyula608/servotimeout/components/script/bluetooth_blacklist.rs:23 #[derive(Eq)]
                                                                              ^~~~~~~~~~~~~
/home/zakor1gyula608/servotimeout/components/script/bluetooth_blacklist.rs:23:1: 23:14 note: in this expansion of #[derive_Eq] (defined in /home/zakor1gyula608/servotimeout/components/script/bluetooth_blacklist.rs)
/home/zakor1gyula608/servotimeout/components/script/bluetooth_blacklist.rs:23:1: 23:14 help: run `rustc --explain E0277` to see a detailed explanation
/home/zakor1gyula608/servotimeout/components/script/bluetooth_blacklist.rs:23:1: 23:14 note: required by `core::cmp::Eq`
error: aborting due to previous error
error: Could not compile `script`.
```